app/models/topic_tracking_state.rb

```diff
@@ -577,6 +577,44 @@ def self.trigger_post_read_count_update(post, groups, last_read_post_number, use
     post.publish_change_to_clients!(:read, opts)
   end
 
+  def self.report_count_by_type(user, type:)
+    tag_ids = muted_tag_ids(user)
+    sql =
+      report_raw_sql(
+        topic_id: nil,
+        skip_unread: type == "new",
+        skip_new: type == "unread",
+        skip_order: true,
+        staff: user.staff?,
+        admin: user.admin?,
+        whisperer: user.whisperer?,
+        user: user,
+        muted_tag_ids: tag_ids,
+      )
+    sql = tags_included_wrapped_sql(sql)
+
+    DB.query(
+      sql + "\n\n LIMIT :max_topics",
+      {
+        user_id: user.id,
+        topic_id: nil,
+        min_new_topic_date: Time.at(SiteSetting.min_new_topics_time).to_datetime,
+        max_topics: TopicTrackingState::MAX_TOPICS,
+        user_first_unread_at: user.user_stat.first_unread_at,
+      }.merge(treat_as_new_topic_params),
+    ).count
+  end
+
+  def self.report_totals(user)
+    if (user.new_new_view_enabled?)
```

Comment:
```suggestion
    if user.new_new_view_enabled?
```

Small one but in Ruby we usually don't add brackets around the condition.

Reply:
Yes. Surprising that `stree` doesn't fix it automatically.